### PR TITLE
108 feature poll in browser

### LINKED ISSUE #108: Feature: poll in browser
Add the poll in browser feature

### LINKED ISSUE #109: Translation: Update translation
Add the polish translation

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
- add poll in browser feature
- minor refactor
- bugfix
- add polish translation

### CODE REVIEW

File: twofactor_privacyidea/js/pi-poll-in-browser.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+window.onload = () =>
+{
+    if (piGetValue("pollInBrowser")
+        && piGetValue("pollInBrowserUrl").length > 0
+        && piGetValue("transactionId").length > 0)
+    {
+        piDisableElement("pushButton");
+        let worker;
+        if (typeof (Worker) !== "undefined")
```
Comment (nilsbehlen) [inline twofactor_privacyidea/js/pi-poll-in-browser.js:27]: what is the point of this check?

File: twofactor_privacyidea/templates/settings-admin.php [hunk 2]
```diff
@@ -172,6 +172,17 @@
             send authentication requests to privacyIDEA directly')); ?>
         </label>
         <hr>
+        <input id="pollInBrowser" type="checkbox" class="checkbox">
+        <label for="pollInBrowser">
+            <?php p($l->t('Poll in browser')); ?>
+            <em>
+                <?php p($l->t('Check this, to activate polling for a push token confirmation right from your browser.')) ?>
```
Comment (nilsbehlen) [inline twofactor_privacyidea/templates/settings-admin.php]: no ","

File: twofactor_privacyidea/js/pi-poll-in-browser.js (hunk 1 above)
Comment (lukasmatusiewicz) [reply twofactor_privacyidea/js/pi-poll-in-browser.js:27]: line 27? or above this statement

Comment (nilsbehlen) [reply twofactor_privacyidea/js/pi-poll-in-browser.js:27]: ` if (typeof (Worker) !== "undefined")`

Comment (lukasmatusiewicz) [reply twofactor_privacyidea/js/pi-poll-in-browser.js:27]: checks if the worker is running

File: twofactor_privacyidea/templates/settings-admin.php (hunk 2 above)
Comment (lukasmatusiewicz) [reply twofactor_privacyidea/templates/settings-admin.php]: [e241471](https://github.com/privacyidea/privacyidea-owncloud-app/pull/110/commits/e24147115c6d2cd63694370b85b1e72856967a64)

**Comment by nilsbehlen** [comment]
can be merged if you are ready

**Pull request merged**